base/src/org/compiere/model/POInfo.java

```diff
@@ -245,6 +248,8 @@ private void loadInfo (boolean baseLanguage, String trxName)
 								trxName);
 					}).toList();
 					list.addAll(infoColumns.asJava());
+				}).onFailure(throwable -> {
+					throw new AdempiereException(throwable);
```

Comment:
Hi @e-Evolution, I think in this section we need an error handler to show if there is any error like the syntax error that was caught here but then didn't log the error or throw it back